docs/ref/class-based-views/generic-date-based.txt

```diff
@@ -372,24 +372,27 @@ views for displaying drilldown pages for date-based data.
 
         <p>
             {% if previous_week %}
-                Previous Week: {{ previous_week|date:"F Y" }}
+                Previous Week: {{ previous_week|date:"W Y" }}
             {% endif %}
             {% if previous_week and next_week %}--{% endif %}
             {% if next_week %}
-                Next week: {{ next_week|date:"F Y" }}
+                Next week: {{ next_week|date:"W Y" }}
             {% endif %}
         </p>
 
-    In this example, you are outputting the week number. The default
-    ``week_format`` in the ``WeekArchiveView`` uses  week format ``'%U'``
-    which is based on the United States week system where the week begins on a
-    Sunday. The ``'%W'`` format uses the ISO week format and its week
-    begins on a Monday.  The ``'%W'`` format is the same in both the
-    :func:`~time.strftime` and the :tfilter:`date`.
-
-    However, the :tfilter:`date` template filter does not have an equivalent
-    output format that supports the US based week system. The :tfilter:`date`
-    filter ``'%U'`` outputs the number of seconds since the Unix epoch.
+    In this example, you are outputting the week number. However, please
+    keep in mind that week numbers computed by the :tfilter:`date`
+    template filter with the ``'W'`` format character are not always the
+    same as those computed by :func:`~time.strftime` with the ``'%W'``
+    format string. For example, for year 2015, week numbers output by
+    :tfilter:`date` are higher by 1 compared to those output by
+    :func:`~time.strftime`. Also note that there is no equivalent for the
+    ``'%U'`` :func:`~time.strftime` format string in :tfilter:`date`
+    either. Therefore, you should avoid using :tfilter:`date` to generate
+    URLs for ``WeekArchiveView``.
+
+    The ``week_format`` attribute defaults to ``'%U'``, and the only valid
```

Comment:
Maybe this would be better in the "Notes" section before the example? It seems like we're explaining details in the previous paragraph before some of the basics are stated here.